lib/components/wallet_connect/approve_token_transfer.dart

```diff
@@ -97,29 +108,35 @@ class _ApproveTokenTransferState extends State<ApproveTokenTransfer> {
                       int targetTick = latestTick + defaultTargetTickType.value;
 
```

Comment:
@ahmed-tarek-salem didn't we said that the targetTick will be calculated only when is empty from the dApp request?

Reply:
@sallymoc 
Yes, I thought I should fix this in all places in a separate PR

Reply:
ApproveTokenTransfer doesn't receive the tick, so the logic is fine.